.travis.yml

```diff
@@ -3,11 +3,13 @@ cache: bundler
 sudo: false
 
 before_install: ./travis/before_install
-bundler_args: --without=development
+bundler_args: --with=development
```

Comment:
The `development` group includes gems that might be usable when coding the project. Travis does not need them.

Reply:
I cannot remember exactly if it was a travis failure or if it was because I introduced 'pry' to the spec files.  I also didn't know that the `--with` and `--without` options are version dependent so I think I remember thinking it was easier to also install the development stuff too if we are testing in travis.  No matter, i'll revert it back as requested.   Thanks.

Reply:
I think the reason was `pry`. It is transitive dependency for `guard-rspec`, which is in the `development` group. So Travis failed to load it.

Reply:
Reverted back to use `--without=development`